probe_scraper/scrapers/moz_central_scraper.py

```diff
@@ -329,15 +331,17 @@ def scrape_channel_revisions(folder=None, min_fx_version=None, max_fx_version=No
 
         print("\nRetreiving Buildhub results for channel " + channel)
 
-        revision_dates = bh.get_revision_dates(channel, min_fx_version, max_version=max_fx_version)
+        revision_dates = bh.get_revision_dates(channel, min_fx_version, max_version=max_fx_version,
+                                               verbose=(channel == "beta"))
```

Comment:
Why is this specifically verbose only for beta?

Reply:
Hmm, this shouldn't have been included.